app/services/support/school_induction_tutors/replace.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Support
+  module SchoolInductionTutors
+    class Replace
+      class << self
+        def call(school_id:, email:, full_name:)
+          new(school_id:, email:, full_name:).call
+        end
+      end
+
+      attr_reader :school, :email, :full_name
+
+      def initialize(school_id:, email:, full_name:)
```

Comment:
I wonder if it would be better to use the URN instead of the school_id, since that's what we usually use when mentioning schools?

Reply:
Good plan, swapped over